### PR TITLE
Add haproxy as front for the project's composition

### PR DESCRIPTION
**Pull request by pvalsecc**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/README.rst_tmpl [hunk 1]
```diff
@@ -6,7 +6,7 @@ Read the `Documentation <http://docs.camptocamp.net/c2cgeoportal/>`_
 Checkout
 --------
 
-.. code::
+.. script::
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/README.rst_tmpl]: Why this change?

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.yaml.mako [hunk 2]
```diff
@@ -48,4 +48,13 @@ ${service_defaults('mapserver', 80)}\
 
   geoportal:
     image: ${docker_base}-geoportal:${docker_tag}
-${service_defaults('geoportal', 80, True)}\
+${service_defaults('geoportal', 80)}\
+
+  proxy:
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.yaml.mako]: The folder is name front and the container proxy shouldn't be the same?

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/README.rst_tmpl (hunk 1 above)
Comment (pvalsecc) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/README.rst_tmpl]: You asked me to change it in the demo, no?

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.yaml.mako (hunk 2 above)
Comment (pvalsecc) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.yaml.mako]: I don't understand?

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/README.rst_tmpl (hunk 1 above)
Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/README.rst_tmpl]: To remove the prompt, but it's already done :-)

Comment (pvalsecc) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/README.rst_tmpl]: OK, reverted
